packages/open-lens/package.json

```diff
@@ -35,7 +35,7 @@
     "dev:renderer": "env NODE_ENV=development ts-node ./webpack/dev-server.ts",
     "test:integration": "jest -xyz --runInBand --detectOpenHandles --forceExit --modulePaths=[\"<rootDir>/integration/\"];",
     "build:tray-icons": "ts-node build/generate-tray-icons.ts",
-    "download:binaries": "ts-node build/download_binaries.ts"
+    "download:binaries": "cd node_modules/@k8slens/download-binaries && npm run download -- --package $INIT_CWD/package.json --base-dir $INIT_CWD/binaries/client"
```

Comment:
I think `@k8slens/download-binaries` should have `bin/download-binaries(.cmd)` (registered via package.json `bin`). This way `download-binaries` will be installed to `open-lens` package under `node_modules/.bin/download-binaries`.

See https://docs.npmjs.com/cli/v9/configuring-npm/package-json#bin